src/AutoRest.CSharp/Common/Generation/Writers/FormattableStringHelpers.cs

```diff
@@ -83,8 +83,13 @@ public static FormattableString GetIdentifiersFormattable(this IEnumerable<strin
             return conversion == null ? constantFormattable : $"{constantFormattable}{conversion}";
         }
 
-        public static FormattableString GetConversionFormattable(this Parameter parameter, CSharpType toType)
+        public static FormattableString GetConversionFormattable(this Parameter parameter, CSharpType toType, CodeWriterDeclaration? bodyOverrideVariable = null)
         {
+            if (bodyOverrideVariable != null)
+            {
+                return $"{bodyOverrideVariable}";
+            }
```

Comment:
If we have the converter helper method, we will never need this "override", we could just check the toType and the type of parameter like below:
```
if (TypeFactory.IsDictionary(parameter.Type) && toType.EqualsIgnoreNullable(typeof(RequestContent)))
return $"<the helper method>";
```

Reply:
The point here is to pass in the content variable.